### ISSUE TITLE
Seeing mega-put speed/progress in command line

### ISSUE THREAD

**Issue by paulo27ms** [meganz/MEGAcmd#133]
Would it be possible to have a progress bar for each file being uploaded or at least some output such as "X file has been uploaded." each time a file is successfully upload?
I'd like to gauge the speed at which my task is going so I can estimate the time when it'll finish and currently there doesn't seem to be any way to do so.

**Comment by polmr** [comment]
There's a progress bar in the interactive console when you do a "put". Having something similar for the non-interactive mode is on our TODO list (no estimated time as of yet). 
Currently, the progress of transfers are shown in the server's log. If you are using linux/mac the log default location should be at $HOME/.megaCmd/megacmdserver.log. If you are using Windows, you would need to start the server separately. More info: https://github.com/meganz/MEGAcmd

**Comment by nexxai** [comment]
Yeah it would be fantastic if this could be added to the non-interactive mode, it would be GREATLY appreciated.

**Comment by polmr** [comment]
This will be included in the next release (scheduled for the following weeks)

**Comment by AssRap3r** [comment]
@polmr are we ever going to see an actual speed counter? Progress is useful for an overall summary but it'd be helpful to have a real transfer speed to know whether an upload has got stuck in the slow lane and needs to be restarted.

**Comment by polmr** [comment]
Sure! Yet unimplemented, but thanks for the reminder @AssRap3r .

**Comment by remigio-cloud4erp** [comment]
Please,
I can't generate log to put more than one file.
I upload several files but the log is saved in white
